product.json

```diff
@@ -25,10 +25,9 @@
 	"privacyStatementUrl": "https://privacy.microsoft.com/privacystatement",
 	"telemetryOptOutUrl": "https://github.com/Microsoft/azuredatastudio/wiki/How-to-Disable-Telemetry-Reporting",
 	"urlProtocol": "azuredatastudio-oss",
-	"enableTelemetry": true,
```

Comment:
It used to be that there was a special check at runtime for whether it was a dev build and if it was it didn't send telemetry. That was changed though to instead just rely completely on this flag - by default this is not set for dev builds and then inserted in during the mixin step for product builds.

This will make it easier to then enable telemetry in dev builds for testing if needed since all you have to do is add this flag to the local product.json.